cmd/fs-v1.go

```diff
@@ -1360,6 +1360,11 @@ func (fs *FSObjects) IsCompressionSupported() bool {
 	return true
 }
 
+// IsObjectTaggingSupported returns whether tagging is applicable for this layer.
```

Comment:
I think you can remove these comments. The comment is wrong here, we know that it is returning true here.